packages/office-ui-fabric-react/src/common/_semanticSlots.scss

```diff
@@ -46,11 +46,22 @@ $buttonBackgroundColor: '[theme:buttonBackground, default: #f4f4f4]';
 $buttonBackgroundHoveredColor: '[theme:buttonBackgroundHovered, default: #eaeaea]';
 $buttonBackgroundCheckedColor: '[theme:buttonBackgroundChecked, default: #c8c8c8]';
 $buttonBackgroundCheckedHoveredColor: '[theme:buttonBackgroundCheckedHovered, default: #eaeaea]';
+$buttonBackgroundPressedColor: '[theme:buttonBackgroundPressed, default: #eaeaea]';
 $buttonBorderColor: '[theme:buttonBorder, default: transparent]';
+$buttonBorderDisabledColor: '[theme:buttonBorderDisabled, default: #767676]';
```

Comment:
group all the button disabled slots together (here and elsewhere)
we need a button bg disabled slot too right?